packages/gapic-node-templating/test/get-bootstrap-template-vars.test.ts

```diff
@@ -108,22 +104,17 @@ describe('get bootstrap template vars', () => {
 
   describe('get package name', () => {
     it('should get the distribution name', async () => {
-      const fileRequest = nock('https://api.github.com')
-        .get(
-          '/repos/googleapis/googleapis/contents/google%2Fcloud%2Fkms%2Fv1%2FBUILD.bazel'
-        )
-        .reply(200, {
-          name: 'BUILD.bazel',
-          content: 'content',
-        });
+      const getContentStub = sinon
```

Comment:
One other non blocking nit, what was the reason for switching from nock to sinon here?